src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -29,6 +29,7 @@ class Cookie
     protected $path;
     protected $secure;
     protected $httpOnly;
+    protected $partitioned;
```

Comment:
The new property should be private (as we have no BC reasons to keep it protected because of the older versions having it protected). For instance, `$sameSite` was also added as private.

and the new property should be typed.